test/sdk.test.ts

```diff
@@ -452,6 +454,164 @@ describe('A SDK', () => {
         },
     );
 
+    it('should configure the evaluator with the specified default timeout', async () => {
+        jest.useFakeTimers();
+
+        const query = '1 + 2';
+        const result = 3;
+
+        fetchMock.mock({
+            method: 'GET',
+            matcher: configuration.cidAssignerEndpointUrl,
+            response: '123',
+        });
+
+        fetchMock.mock({
+            method: 'POST',
+            matcher: `begin:${configuration.baseEndpointUrl}`,
+            delay: 200,
+            response: JSON.stringify(result),
+        });
+
+        const sdk = Sdk.init({
+            ...configuration,
+            defaultFetchTimeout: 5,
+        });
+
+        const promise = sdk.evaluator.evaluate(query);
+
+        jest.advanceTimersByTime(6);
+
+        await expect(promise).rejects.toThrowWithMessage(
+            EvaluationError,
+            'Maximum evaluation timeout reached before evaluation could complete.',
+        );
+
+        await expect(promise).rejects.toHaveProperty('response', expect.objectContaining({
+            detail: 'The evaluation took more than 5ms to complete.',
+        } satisfies Partial<EvaluationErrorResponse>));
+    });
+
+    it('should configure the evaluator with the default timeout', async () => {
+        jest.useFakeTimers();
+
+        const query = '1 + 2';
+        const result = 3;
+
+        fetchMock.mock({
+            method: 'GET',
+            matcher: configuration.cidAssignerEndpointUrl,
+            response: '123',
+        });
+
+        fetchMock.mock({
+            method: 'POST',
+            matcher: `begin:${configuration.baseEndpointUrl}`,
+            delay: 10_000,
+            response: JSON.stringify(result),
+        });
+
+        const {defaultFetchTimeout: _, ...sdkConfiguration} = configuration;
+
+        const sdk = Sdk.init(sdkConfiguration);
+
+        const promise = sdk.evaluator.evaluate(query);
+
+        jest.advanceTimersByTime(5_001);
```

Comment:
It is just a detail, but the constant default timeout could be imported and summed to 1 so the test stays updated regardless of the value change.

Reply:
That's a good point, but maybe that const shouldn't be exported, otherwise we're are exposing it to the clients without a clear reason other than testing

If the default value increases, the test stop working, IMO this is a good enough feedback

Reply:
The container is inaccessible from the userland, so exporting this for external awareness is okay.